stringtable.xml

```diff
@@ -0,0 +1,6722 @@
+			<Spanish>Refuerzos hostiles llamados:</Spanish>
+			<Russian>Враждебных подкреплений вызвано:</Russian>
+			<Italian>Chiamati rinforzi ostili:</Italian>
+			<Chinesesimp>敌军增援次数：</Chinesesimp>
+			<Chinese>敵方增援次數：</Chinese>
+			<Turkish>Düşman takviye kuvvetleri:</Turkish>
+			<Portuguese>Reforços hostis requisitados:</Portuguese>
+			<Polish>wezwanych wrogich posiłków:</Polish>
+		</Key>
+		<Key ID="STR_STATS_21">
+			<Original>Total combat readiness raised:</Original>
+			<French>Augmentation totale du niveau d'alerte :</French>
+			<German>Gesamte Kampfbereitschaft erhöht:</German>
+			<Spanish>Disposición de combate total recaudada:</Spanish>
+			<Russian>Всего войск поднято по тревоге:</Russian>
+			<Italian>Disposizione al combattimento recuperata:</Italian>
+			<Chinesesimp>总备战比例：</Chinesesimp>
+			<Chinese>威脅度比例：</Chinese>
+			<Turkish>Hazırda olan asker sayısı:</Turkish>
+			<Portuguese>Disposição total de combatentes em alerta:</Portuguese>
+			<Polish>podniesionych gotowości bojowych w sumie:</Polish>
+		</Key>
+		<Key ID="STR_STATS_22">
+			<Original>IEDs detonated:</Original>
+			<French>Explosions d'IED :</French>
+			<German>IEDs zur Explosion gebracht:</German>
+			<Spanish>IEDs detonados:</Spanish>
+			<Russian>Фугасов взорвано:</Russian>
+			<Italian>IED detonato:</Italian>
+			<Chinesesimp>被引爆的IED：</Chinesesimp>
+			<Chinese>被引爆的IED：</Chinese>
+			<Turkish>Patlatılan mayınlar:</Turkish>
+			<Portuguese>IEDs detonadas:</Portuguese>
+			<Polish>zdetonowanych min-pułapek:</Polish>
+		</Key>
+		<Key ID="STR_STATS_23">
+			<Original>Number of Spartan 01 losses:</Original>
+			<French>Pertes de Spartan 01 :</French>
+			<German>Höhe der Spartan-01 Verluste:</German>
+			<Spanish>Número de pérdidas de Spartan 01:</Spanish>
+			<Russian>Количество потерянных Spartan 01:</Russian>
+			<Italian>Numero di Spartan 01 persi :</Italian>
+			<Chinesesimp>损失的斯巴达01号：</Chinesesimp>
+			<Turkish>Spartan 01 kayıpları:</Turkish>
+			<Portuguese>Número de baixas de Spartan 01:</Portuguese>
+			<Polish>suma strat Spartana 01:</Polish>
+		</Key>
+		<Key ID="STR_STATS_24">
+			<Original>Player deaths:</Original>
+			<French>Joueurs morts :</French>
+			<German>Spielertode:</German>
+			<Spanish>Muertes del jugador:</Spanish>
+			<Russian>Смертей игрока:</Russian>
+			<Italian>Giocatori Deceduti:</Italian>
+			<Chinesesimp>玩家死亡数：</Chinesesimp>
+			<Chinese>玩家死亡數：</Chinese>
+			<Turkish>Ölen oyuncular:</Turkish>
+			<Portuguese>Morte de jogadores:</Portuguese>
+			<Polish>zgonów gracza:</Polish>
+		</Key>
+		<Key ID="STR_STATS_25">
+			<Original>Rabbits killed:</Original>
+			<French>Lapins tués :</French>
+			<German>Hasen getötet:</German>
+			<Spanish>Conejos asesinados:</Spanish>
+			<Russian>Кроликов убито:</Russian>
+			<Italian>Conigli uccisi:</Italian>
+			<Chinesesimp>兔子屠杀数：</Chinesesimp>
+			<Chinese>兔子屠殺數：</Chinese>
+			<Turkish>Öldürülen tavşanlar:</Turkish>
+			<Portuguese>Coelhos mortos:</Portuguese>
+
```

Comment:
`Czy jesteś pewien że chcesz przepakować to FOB?` bardziej oddaje sens oryginału.